tgui/packages/tgui/interfaces/CrewConsoleSkyrat.js

```diff
@@ -44,9 +43,7 @@ const jobToColor = (jobId) => {
   if (jobId >= 60 && jobId < 80) {
     return COLORS.department.service;
   }
-  if (jobId >= 200 && jobId < 240) {
-    return COLORS.department.centcom;
-  }
+
```

Comment:
```suggestion
  if (jobId >= 200 && jobId < 240) {
    return COLORS.department.centcom;
  }
```
Might as well just put it back for now then.

Reply:
Oh you just did as I was writing this. Nice!